hooks/useNWACObservations.ts

```diff
@@ -17,30 +17,34 @@ import {ZodError} from 'zod';
 export const useNWACObservations = (center_id: AvalancheCenterID, endDate: RequestedTime) => {
   const {nwacHost} = React.useContext<ClientProps>(ClientContext);
   const {logger} = React.useContext<LoggerProps>(LoggerContext);
-  const key = queryKey(nwacHost, center_id);
+  const key = queryKey(nwacHost, center_id, endDate);
```

Comment:
the `queryKey` function was ignoring dates, so it was impossible to query for observations in a different date range - you'd always get whatever preloaded observations had been cached. 

note that `endDate` can be `'latest'` here, which is also what the preloaded query uses. that means that when viewing the latest info, we'll get the preloaded obs from the cache even though there's a delta between the time of preloading and the actual time that the obs screen is viewed.

Reply:
Good catch. I think this is the behavior we want anyway, that's the point of `'latest'` - if you're trying to get the latest, we want to cache something for you but we can't use the current time.